ts/features/wallet/bancomat/screen/BancomatInformation.tsx

```diff
@@ -21,7 +19,7 @@ import { GlobalState } from "../../../../store/reducers/types";
 
 type OwnProps = {
   onAddPaymentMethod?: () => void;
-  hideInfobox?: boolean;
+  hideCobrandTitle?: boolean;
```

Comment:
```suggestion
  hideCobrandTitle?: true;
```
Maybe it could be `true | undefined `. When you need `hideCobrandTitle=false` you could use undefined or simply omit that prop